client/packages/requisitions/src/ResponseRequisition/DetailView/ResponseLineEdit/ResponseLineEdit.tsx

```diff
@@ -69,6 +73,11 @@ export const ResponseLineEdit = ({
       ? available / (draft?.averageMonthlyConsumption ?? 1)
       : 0;
 
+  const customRequestedDisabled =
+    !!hasLinkedRequisition ||
+    (!!isProgram && reasonsData?.nodes.length === 0) ||
+    reasonsLoading;
```

Comment:
Why have you disabled requested with these extra conditions? Users should be able to input requested quantity if the Requisition hasn't been generated from an Internal Order